### PR TITLE
Feature/title option

### PR DESCRIPTION
**Pull request by dhulme**
Tested locally and confirmed works with + without `title` option

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,6 +160,9 @@ cy.matchImage({
   // helps with screenshots being scaled 2x on high-density screens like Mac Retina
   // default: true
   forceDeviceScaleFactor: false,
+  // image title
```
Comment (FRSgit) [inline README.md:163]:
```suggestion
  // title used for naming the image file
```

**Review by FRSgit — CHANGES_REQUESTED** [review]
Good PR, thank you 🙏 
I have one comment about the docs wording and it's good to be merged!

**Comment by dhulme** [comment]
Thanks! Done 👍

**Review by FRSgit — APPROVED** [review]
Now it's perfect, I'll merge it down (want to make sure the commit message is using `conventional-commits` pattern - otherwise it won't trigger autorelease 😄

**Comment by FRSgit** [comment]
By the way - thanks for first PR on this repo. You've make one CI bug visible - now I could've fixed it on main branch! 😄 🎉

**Comment by FRSgit** [comment]
Released in 1.9.0

**Pull request merged**